config/crd/bases/risingwave.risingwavelabs.com_risingwaves.yaml

```diff
@@ -879,6 +879,33 @@ spec:
                   object:
                     description: Storage spec for object storage.
                     properties:
+                      aliyunOSS:
```

Comment:
Thanks for adding these descriptions. I always use `explain` to understand  a field and this is really helpful 👍 

```
k explain rw.spec.storages.object.aliyunOSS
KIND:     RisingWave
VERSION:  risingwave.risingwavelabs.com/v1alpha1

RESOURCE: aliyunOSS <Object>

DESCRIPTION:
     AliyunOSS storage spec.

FIELDS:
   bucket	<string> -required-
     Bucket of the Aliyun OSS service.

   internalEndpoint	<boolean>
     InternalEndpoint indicates if we use the internal endpoint to access Aliyun
     OSS, which is only available in the internal network.

   region	<string>
     Region of Aliyun OSS service. It is an optional field that overrides the
     `Region` key from the secret. Specifying the region here makes a guarantee
     that it won't be changed anymore.

   secret	<string> -required-
     Secret contains the credentials to access the Aliyun OSS service. It must
     contain the following keys: * AccessKeyID * SecretAccessKey * Region
     (optional if region is specified in the field.)
```